src/macros.jl

```diff
@@ -456,6 +471,9 @@ function parse_constraint(_error::Function, args...)
           "       expr1 == expr2\n" * "       lb <= expr <= ub")
 end
 
+function_to_set(_error::Function, ::Val{:alldifferent}) = nothing # TODO: Needs a MOI extension (new set alldifferent).
```

Comment:
Why is this method needed ?

Reply:
It is there to make the translation between the syntax offered by JuMP and the sets that will be defined on MOI's side (or rather an extension, I suppose), in the same spirit as `sense_to_set`.